examples/scripts/ppo.py

```diff
@@ -174,7 +175,7 @@ def collator(data):
     "max_new_tokens": 32,
 }
 
-for _epoch, batch in tqdm(enumerate(ppo_trainer.dataloader)):
+for _epoch, batch in tqdm(enumerate(ppo_trainer.dataloader), total=len(ppo_trainer.dataloader)):
```

Comment:
`total` is set to `len(iterable)` by default, why do you need this?

Reply:
Thanks for the review. When I ran the original code, I didn't see the total number of iterations getting printed. My understanding is that `enumerate` returns a generator in which case `len()` wouldn't work. Alternatively, we can probably just do `tqdm(ppo_trainer.dataloader)`.

Reply:
Sounds good, thanks for the patch